### PR TITLE
Make changes to the algorithim

### PR DESCRIPTION
**Pull request by spaceyspacestar**
I made a change that'll boost my ego than the smelly wario cosplayer Elon.
...What? If Elon Musk can do it, Why can't I as well?

### CODE REVIEW

**Comment by spaceyspacestar** [comment]
looks good to me

**Review by thegu5 — COMMENTED** [review]
LGTM

**Review by apal0934 — COMMENTED** [review]
LGTM!

**Comment by Cyborus04** [comment]
LGTM, critical change IMO

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala [hunk 1]
```diff
@@ -199,6 +199,7 @@ object HomeTweetTypePredicates {
       candidate =>
         candidate.getOrElse(EarlybirdFeature, None).exists(_.hasImage) &&
           !candidate.getOrElse(HasDisplayedTextFeature, false)),
+    ("author_is_spacey, _.getOrElse(AuthorIdFeature, None).contains(candidate.getOrElse(DDGStatsSpaceyFeature, 0L))),
```
Comment (SEQUOiA87) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:202]: It seems there is a double quote missing here after `"author_is_spacey`.

Comment (spaceyspacestar) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala]: sorry i fixed it, I was rushing trying to write this lol

**Comment by BeaNoodle** [comment]
I agree 👍

**Pull request closed without merging**